From fec76d46982d2001d673e95598bde15df0986d07 Mon Sep 17 00:00:00 2001
From: F4B <no@contact.com>
Date: Sat, 1 Oct 2022 10:22:42 +0200
Subject: [PATCH] fix: resolve undeclared symbol compilation error

---
 drivers/bluetooth/bcm43xx.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/bluetooth/bcm43xx.c b/drivers/bluetooth/bcm43xx.c
index 9cf8219d634e..e726d0265ef4 100644
--- a/drivers/bluetooth/bcm43xx.c
+++ b/drivers/bluetooth/bcm43xx.c
@@ -77,8 +77,6 @@ struct bcm_bt_gpio {
 
 int idle_ip_index;
 
-EXPORT_SYMBOL(check_bt_op);
-
 static int bcm43xx_bt_rfkill_set_power(void *data, bool blocked)
 {
 	/* rfkill_ops callback. Turn transmitter on when blocked is false */
